### DISCUSSION TITLE
Lists are not properly supporting markdown formatting when checkboxes are used

### DISCUSSION THREAD

**Discussion by Fogest** [requarks/wiki#4752] [Error / Bug Report] [answered]
I can't seem to get lists to work with formatting such as bolding, links, etc...

See image below as an example:

![image](https://user-images.githubusercontent.com/1163628/144688142-0dc16784-ed36-40c3-98d8-c6cca2f1fd7d.png)

If I add another character after the link it changes how it formats it too:

![image](https://user-images.githubusercontent.com/1163628/144688170-90b1dedd-bfd0-459a-add8-7819fc91a756.png)

And it is not just the preview pane that is messed up, it'll look like this on the rendered page as well. If I remove the checkmarks it works fine. It's just when using the todo list that it breaks.

For now I'll probably just use an emoji checkmark and an X emoji to indicate things being done or not, but I'd rather use the native method. 

I am using the latest public release of v2.5.219 via the docker setup.

**Comment by NGPixel** [comment, marked as answer]
Fixed by 0d7d95d2ac569149e637271f7bae61996b327a40

You can use `requarks/wiki:canary-2.5.257` if using docker to get the fix now. Otherwise it will be included in the next release.

**Comment by Fogest** [comment]
Awesome, thank you so much!

I'm not using docker currently for this setup so I'll probably just wait
until the next release to give it a shot :).

Appreciate the effort on this project!
